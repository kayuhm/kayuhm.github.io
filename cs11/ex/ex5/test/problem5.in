2
10
36 25 38 46 55 68 72 55 36 38 67 45 22 48 91 46 52 61 58 55
50
2 2 1 255 0 3 4 5 6 7 200 100 10 50 0