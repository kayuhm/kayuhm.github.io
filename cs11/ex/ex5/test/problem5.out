22-31 2
32-41 4
42-51 4
52-61 6
62-71 2
72-81 1
82-91 1

0-49 11
50-99 1
100-149 1
150-199 0
200-249 1
250-299 1

